ejercicio1vulky/src/main/java/com/mycompany/ejercicio1vuky/Persona.java

```diff
@@ -0,0 +1,30 @@
+package com.mycompany.ejercicio1vuky;
+
+public class Persona implements AccionPersona {
+
+    public ModeloPersona informacionGlobal;
+
+    public void llenarInformacion(ModeloPersona informacionlocal) {
+        informacionGlobal = informacionlocal;
+
+    }
+
+    @Override
+    public void caminar() {
+    }
+
+    @Override
+    public void saltar() {
```

Comment:
Please add a functionality in the method

Reply:
ready.